app/docker/views/services/edit/service.html

```diff
@@ -71,6 +71,24 @@
                     ng-model="service.Image" ng-change="updateServiceAttribute(service, 'Image')" id="image_name" ng-disabled="isUpdating">
               </td>
             </tr>
+            <tr>
+              <td colspan="{{webhookURL ? '1' : '2'}}">
+                Update webhook
```

Comment:
"Update webhook" here sounds more like we are asking to update the webhook instead of taking this whole thing as a noun. Should we change it to something like "Service webhook" or "Update service via webhook"?

Reply:
Yes, let's switch to Service webhook here.